src/components/EntityInfo.vue

```diff
@@ -44,7 +44,7 @@
                   @click="promptChangeCompanyInfo()"
                 >
                   <v-icon medium>mdi-file-document-edit-outline</v-icon>
-                  <span class="font-13 ml-1">View and Change Company Information</span>
+                  <span class="font-13 ml-1">View and Change Business Information</span>
```

Comment:
This change affects all entity types. Is it approved by UI/UX team?

Reply:
Yeah, UI/UX team approved it

<img width="943" alt="Screen Shot 2022-05-27 at 3 01 15 PM" src="https://user-images.githubusercontent.com/55506249/170794477-be394e0e-cffd-4fd1-888c-a22837520970.png">

Reply:
Woohoo! (I've been asking for this for months.)